.appveyor.yml

```diff
@@ -0,0 +1,23 @@
+build: false
+
+platform:
+  - x64
+
+environment:
+  matrix:
+    # Python 3.6
```

Comment:
I think the latest or possibly upcoming miniconda is 3.7 I would say leave the comment out since it will soon be outdated if it is not already